### ISSUE TITLE
docs: create CONTRIBUTING.md file

### ISSUE THREAD

**Issue by bitfalt** [ZencypherSolutions/semaphore-stellar-docs#3]
Create a `CONTRIBUTING.md` explaining the process to create an issue, and fork the repo. Also, you must include best practices for naming, since this is a docs repository. The name for issues can be docs or research. 

Any additional suggestions are welcome!

**Comment by Jemiiah** [comment]
@bitfalt I would love to work on this issue

**Comment by 0xBinayak** [comment]
@bitfalt I am a  new contributer and have contributed to may repos i think i can make a CONTRIBUTING.md with  very easy and clear instructions.
 Time frame :under 1 Day

**Comment by KevinLatino** [comment]
I am a Dojo Coding Member

I would like to take this issue. I will create a CONTRIBUTING.md file that outlines the following:

Process to Create an Issue: A step-by-step guide.
Forking the Repository: Instructions for forking and cloning.
Best Practices for Naming: Recommendations for issue naming, focusing on clarity and categorization (e.g., docs, research).

I’m also open to any additional suggestions you may have!

**Comment by PoulavBhowmick03** [comment]
I’d like to help with this.

I will do as asked, create a CONTRIBUTING.md, I will create and write proper instructions on how to create an issue, fork the repo and for the best practices
I will also add a template for users to use to create a PR, and other resources for them to refer to for various sections

ETA- 12 hours

**Comment by JosueBrenes** [comment]
Hi, I'm **Josué Brenes** and I'll be working on issue.
I'm Dojo Coding member.

I estimate this will take **1 day** to complete.

## How I would resolve this

The CONTRIBUTING.md file should provide clear guidelines for contributing to the repository. 
For example you can carry the following

### Creating an Issue

1. Go to the **Issues** tab.
2. Click on **New Issue**.
3. Choose a title format for the issue:
   - For documentation-related issues, use the prefix `docs`.
   - For research-related issues, use the prefix `research`.
4. Provide a detailed description of the issue, including any relevant context, screenshots, or error messages.
5. Submit the issue.

### Forking the Repository

1. Click on the **Fork** button in the top right corner of the repository page.
2. Clone your forked repository to your local machine:
   ```bash
   git clone https://github.com/your-username/repo-name.git
   ```
3. Create a new branch for your contribution:
   ```bash
   git checkout -b feature/your-feature-name
   ```
4. Make your changes, commit them, and push the branch to your fork:
   ```bash
   git add .
   git commit -m "Description of your changes"
   git push origin feature/your-feature-name
   ```

### Submitting a Pull Request

1. Go to the original repository.
2. Click on **Pull Requests** and select **New Pull Request**.
3. Choose the branch from your fork and create the pull request.
4. Provide a clear and descriptive title and description for the pull request.

### Naming Best Practices

- Use clear and descriptive names for branches and issues.
- For issues, use the prefix `docs` or `research` to specify the type of contribution.
- For branches, follow this format: `feature/description` or `fix/description`.

---

**Comment by NueloSE** [comment]
I’d love to work on this task.

**Comment by josephpdf** [comment]
Hi!, my name is Joseph Poveda.
I am a student of the Systems Programming degree, I am a recent member of the Dojo Coding community, where I am getting started in the world of Web3.
To approach this issue, I would first create a CONTRIBUTING.md file explaining the steps for creating an issue and forking the repository. I’d include naming best practices, such as using "docs" or "research" as prefixes to categorize content types. Additionally, I’d offer general advice on organizing documentation and summarize collaboration best practices, especially for documentation repositories, while providing extra tips on structuring and clarity for documentation.

**Comment by petersdt** [comment]
I'd love to work on this!

**Comment by martinvibes** [comment]
I'd like to take this issue.
please kindly assign :) and i'll get to work asap

## Issue Summary for CONTRIBUTING.md Creation
**Objective:** Develop a CONTRIBUTING.md file to guide contributors on repository processes.

#### Details:

- Outline steps for creating an issue and forking the repository.
- Include naming conventions for issues (e.g., starting with docs or research).
- Incorporate best practices for contribution structure, especially relevant to this documentation-focused repository.

ETA: 24HRS

**Comment by derianrddev** [comment]
Hello, I’m Derian, a software engineer focused on full-stack web development and currently part of Dojo Coding. I’d like to work on this issue.

How I would resolve the issue:

Create the CONTRIBUTING.md file with the sections described in the issue, along with any additional suggestions. 
Save and upload the file to the repository.
If I have questions, I’ll ask them as soon as possible in the Telegram group. 
I will open a PR and wait for your feedback. 
Make any necessary changes according to the PR review.

**Comment by deeseeker** [comment]
Could I take on this issue?

**Comment by jsandinoDev** [comment]
Hi @bitfalt I'm a full-stack developer with web3 experience member of Dojo Coding, specializing in smart contracts and decentralized applications. I've worked on projects using React, Solidity, Cairo, and blockchain tools. My skills in both front-end and back-end development allow me to build scalable and secure solutions for web3 ecosystems. I have worked on modifying the readme in projects like Shinigami & GoStarkme
I would love to work on this issue
